From: losuler <losuler@posteo.net>
Subject: Use offline sources from OBS
--- a/makeicecat	2021-11-13 12:41:55.346614456 +1100
+++ b/makeicecat	2021-11-13 12:44:27.631453583 +1100
@@ -119,16 +119,6 @@
         return 1
     fi
 
-    # verify that Mercurial is available
-    if ! which hg &> /dev/null
-    then
-        echo -e "\nERROR: This script requires Mercurial:
- e.g.: 'hg' from the Guix 'mercurial' package
-       'hg' from the Parabola 'mercurial' package
-       'hg' from the Trisquel 'mercurial' package"
-        return 1
-    fi
-
     # verify that Perl 'rename' is available and sufficent
     local rename_cmds=(
         # NOTE: order is important -- 'util-linux' provides a binary named 'rename'
@@ -155,16 +145,6 @@
         return 1
     fi

-    # verify that Wget is available
-    if ! which wget &> /dev/null
-    then
-        echo -e "\nERROR: This script requires Wget:
- e.g.: 'wget' from the Guix 'wget' package
-       'wget' from the Parabola 'wget' package
-       'wget' from the Trisquel 'wget' package"
-        return 1
-    fi
-
     # verify that Python 3 is available
     if ! which python3 &> /dev/null
     then
@@ -223,7 +203,7 @@
 verify_sources()
 {
     echo Verifying Firefox tarball
-    gpg --recv-keys --keyserver keyserver.ubuntu.com ${SOURCEBALL_SIGNINGKEY}
+    gpg --import KEY
     gpg --verify ${SOURCEBALL}.asc
     echo -n ${SOURCEBALL_CHECKSUM} ${SOURCEBALL} | sha256sum -c -
 }
@@ -250,7 +230,8 @@
         if [[ -n "${lang}" && "${lang}" != 'en-US' && ! -f ${lang}/${L10N_DTD_FILE} ]]
         then
             echo "Cloning '${lang}' language repository"
-            hg clone ${L10N_URL}/${lang}
+            unzip ../$lang.zip
+            find . -mindepth 1 -name "$lang-*" -prune -type d -exec mv {} $lang \;
             mkdir -p ${lang}/${L10N_PREFS_DIR}
             touch ${lang}/${L10N_DTD_FILE}
             rm -rf ${lang}/.hg*
@@ -259,13 +240,13 @@
     cd ..
     cp -a l10n ${SOURCEDIR}/

+    unzip compare-locales-${L10N_CMP_REV}.zip
+    mv compare-locales-${L10N_CMP_REV} compare-locales
+
     if ! [[ -d compare-locales ]]
     then
         hg clone ${L10N_CMP_URL}/
     fi
-    cd compare-locales/
-    hg checkout ${L10N_CMP_REV}
-    cd ..
     cp -a compare-locales ${SOURCEDIR}/l10n/
     rm -rf ${SOURCEDIR}/l10n/compare-locales/{.git,.hg}*
 }
