--- a/makeicecat	2021-02-15 12:56:03.091078013 +1100
+++ b/makeicecat	2021-02-15 15:25:26.389348044 +1100
@@ -119,16 +119,6 @@
         return 1
     fi
 
-    # verify that Mercurial is available
-    if ! which hg &> /dev/null
-    then
-        echo -e "\nERROR: This script requires Mercurial:
- e.g.: 'hg' from the Guix 'mercurial' package
-       'hg' from the Parabola 'mercurial' package
-       'hg' from the Trisquel 'mercurial' package"
-        return 1
-    fi
-
     # verify that Perl 'rename' is available and sufficent
     local rename_cmds=(
         # NOTE: order is important -- 'util-linux' provides a binary named 'rename'
@@ -154,16 +144,6 @@
        'prename' from the Trisquel 'rename' package"
         return 1
     fi
-
-    # verify that Wget is available
-    if ! which wget &> /dev/null
-    then
-        echo -e "\nERROR: This script requires Wget:
- e.g.: 'wget' from the Guix 'wget' package
-       'wget' from the Parabola 'wget' package
-       'wget' from the Trisquel 'wget' package"
-        return 1
-    fi
 }
 
 prepare_env()
@@ -203,7 +183,7 @@
 verify_sources()
 {
     echo Verifying Firefox tarball
-    gpg --recv-keys --keyserver keyserver.ubuntu.com ${SOURCEBALL_SIGNINGKEY}
+    gpg --import KEY
     gpg --verify ${SOURCEBALL}.asc
     echo -n ${SOURCEBALL_CHECKSUM} ${SOURCEBALL} | sha256sum -c -
 }
@@ -230,7 +210,8 @@
         if [[ -n "${lang}" && "${lang}" != 'en-US' && ! -f ${lang}/${L10N_DTD_FILE} ]]
         then
             echo "Cloning '${lang}' language repository"
-            hg clone ${L10N_URL}/${lang}
+            unzip ../$lang-[a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9].zip
+            find . -mindepth 1 -name "$lang-*" -prune -type d -exec mv {} $lang \;
             mkdir -p ${lang}/${L10N_PREFS_DIR}
             touch ${lang}/${L10N_DTD_FILE}
             rm -rf ${lang}/.hg*
@@ -239,13 +220,13 @@
     cd ..
     cp -a l10n ${SOURCEDIR}/
 
+    unzip compare-locales-${L10N_CMP_REV}.zip
+    mv compare-locales-${L10N_CMP_REV} compare-locales
+
     if ! [[ -d compare-locales ]]
     then
         hg clone ${L10N_CMP_URL}/
     fi
-    cd compare-locales/
-    hg checkout ${L10N_CMP_REV}
-    cd ..
     cp -a compare-locales ${SOURCEDIR}/l10n/
     rm -rf ${SOURCEDIR}/l10n/compare-locales/{.git,.hg}*
 }
